packages/generator/src/generator-without-ts-morph/service/class.spec.ts

```diff
@@ -24,27 +24,20 @@ describe('class', () => {
     expect(result).not.toContain('actionImports');
   });
 
-  it('contains functionImports if in VDM', () => {
+  it('contains operations if in VDM', () => {
     const result = serviceClass({
       ...service,
       functionImports: [
         { name: 'myFunction', parametersTypeName: 'paraName' } as any
+      ],
+      actionImports: [
+        { name: 'myAction', parametersTypeName: 'paraName' } as any
       ]
     });
-    expect(result).toContain('functionImport');
+    expect(result).toContain('operations');
```

Comment:
I think this is because `serviceClass(...)` returns a string that contains all the `service` code (I just found out this now), and this `expect...` line basically tried to see if the generated `service` code contained the substring `functionImport`, which it did.